config/crd/bases/slo.koordinator.sh_nodeslos.yaml

```diff
@@ -1222,6 +1222,59 @@ spec:
                       cpuPolicy:
                         description: applied policy for the CPU QoS, default = "groupIdentity"
                         type: string
+                      networkQOS:
```

Comment:
why add this field? didn't see it referenced anywhere.

Reply:
Added based on the previous comments https://github.com/koordinator-sh/koordinator/pull/1843#discussion_r1470551137

Reply:
I think this means you should implement `netqospolicy` in a similar way to `cpupolicy`, something like this
```golang
type ResourceQOSPolicies struct {
	// applied policy for the CPU QoS, default = "groupIdentity"
	CPUPolicy    *CPUQOSPolicy `json:"cpuPolicy,omitempty"`
	// applied policy for the Net QoS, default = "tc"
	NETQOSPolicy *NETQOSPolicy `json:"NETQOSPolicy,omitempty"`
}

type NETQOSPolicy string

const (
	// NETQOSPolicyTerwayQos indicates implement netqos by terway-qos.
	NETQOSPolicyTerwayQos NETQOSPolicy = "terway-qos"
	// NETQOSPolicyTC indicates implement netqos by tc.
	NETQOSPolicyTC NETQOSPolicy = "tc"
)
```

Reply:
If that's the understanding, does a runtime plugin need to support multiple implementations? 

Previously, I thought that by using a runtime plugin, different implementations were supported through multiple runtime plugins.